docs: update readme (#202)
Layr-Labs/avs-devnet: README.md

Note that only one devnet per file name can be running at the same time.
Trying to start another one (or the same one more than once) will fail.

> [!TIP]

> > If you're having any problems running the devnet, check the ["Troubleshooting"](#troubleshooting) section for known issues.

Reviewer: ```suggestion
> If you encounter any issues while running the software the devnet, check the ["Troubleshooting"](#troubleshooting) section for known problems.
```

Author: Done
